lib/apollo-federation/schema.rb

```diff
@@ -29,6 +37,10 @@ def federation_2?
         Gem::Version.new(federation_version.to_s) >= Gem::Version.new('2.0.0')
       end
 
+      def federation_2_1?
```

Comment:
I don't think we should add specific minor version support. We do have some cleanup to do, but it's going to be easier to maintain this lib if we simply aim to support the latest version of federation instead of incrementally supporting minor versions of 2.x. See [this issue](https://github.com/Gusto/apollo-federation-ruby/issues/221) for more context.